### DISCUSSION TITLE
是否有计划接入诸如Auth0等鉴权工具来保护在线网页？

### DISCUSSION THREAD

**Discussion by cnkjw** [ChatGPTNextWeb/NextChat#2459] [Ideas]
当前ChatGPT-Next-Web自带的保护机制有点弱，有被爆破的风险。是否可以提供简单的方案接入Auth0类似的鉴权工具，以便保护个人部署不被滥用。

**Comment by lloydzhou** [comment]
这个你可以在外层的网关上接入一个，做鉴权
